pkg/apis/camel/v1/camelcatalog_types.go

```diff
@@ -75,6 +75,7 @@ type CamelCatalogStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Runtime Version",type=string,JSONPath=`.spec.runtime.version`,description="The Camel K Runtime version"
 // +kubebuilder:printcolumn:name="Runtime Provider",type=string,JSONPath=`.spec.runtime.provider`,description="The Camel K Runtime provider"
+// +kubebuilder:resource:path=integrations,scope=Namespaced,shortName=it,categories=kamel;camel
```

Comment:
ditto :)

Reply:
done, thanks :)

Reply:
I still see the duplicated marker.

Reply:
dah, my mistake on the commit, sorry, if you please check now